internal/pkg/api/handleCheckin.go

```diff
@@ -376,6 +376,10 @@ func convertActions(agentID string, actions []model.Action) ([]ActionResp, strin
 
 	respList := make([]ActionResp, 0, sz)
 	for _, action := range actions {
+		if action.Type == "POLICY_CHANGE" {
+			log.Warn().Str("agent_id", agentID).Str("action_id", action.ActionID).Msg("Removing POLICY_CHANGE action found in index from check in response")
```

Comment:
Question: When are these policy change actions an issue on check in?

Looking at the code, it seems that `TypePolicyChange` actions are only created on checkin code, in the `processPolicy` method, so it doesn't seem to be always an issue on checkin.

Here a filter is being added to `convertAction`, that is used when:
* There are pending actions.
* There are not pending actions, but something is received from the actions dispatcher.

Are these policy change actions an issue on both cases?
Are they generated in some other place too?

If this checkin handler is the only source of these actions, there would be some way to avoid generating them instead of having to filter them later?

Reply:
We only want to filter out policy_change actions if the actions are read from the index, not if fleet-server generates them by detecting a different policy

Reply:
Thanks for the explanations!

> the actions index 

> It's possible for a user to insert a policy change action this way.

How can actions be added to the actions index? Or how can users insert policy changes?